[2024-12-19T07:08:52.482+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T07:08:52.494+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2021-10-26T00:00:00+00:00 [queued]>
[2024-12-19T07:08:52.500+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2021-10-26T00:00:00+00:00 [queued]>
[2024-12-19T07:08:52.501+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T07:08:52.512+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): second_task> on 2021-10-26 00:00:00+00:00
[2024-12-19T07:08:52.518+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=7886) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T07:08:52.518+0000] {standard_task_runner.py:72} INFO - Started process 7895 to run task
[2024-12-19T07:08:52.518+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'second_task', 'scheduled__2021-10-26T00:00:00+00:00', '--job-id', '1424', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp_m_hlisf']
[2024-12-19T07:08:52.521+0000] {standard_task_runner.py:105} INFO - Job 1424: Subtask second_task
[2024-12-19T07:08:52.554+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v1.second_task scheduled__2021-10-26T00:00:00+00:00 [running]> on host 13c961a72918
[2024-12-19T07:08:52.592+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='code2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-10-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-10-26T00:00:00+00:00'
[2024-12-19T07:08:52.594+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T07:08:52.602+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T07:08:52.603+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1']
[2024-12-19T07:08:52.607+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T07:08:52.607+0000] {subprocess.py:106} INFO - hey, I am task2 and will be running after task1
[2024-12-19T07:08:52.607+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T07:08:52.619+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T07:08:52.620+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=second_task, run_id=scheduled__2021-10-26T00:00:00+00:00, execution_date=20211026T000000, start_date=20241219T070852, end_date=20241219T070852
[2024-12-19T07:08:52.660+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T07:08:52.666+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T07:08:52.668+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
